lib/prepare.js

```diff
@@ -378,14 +378,18 @@ function updateProjectSplashScreen (platformConfig, locations) {
         'windowSplashScreenAnimatedIcon',
         'windowSplashScreenAnimationDuration',
         'windowSplashScreenBackground',
-        'windowSplashScreenBrandingImage',
+        'android:windowSplashScreenBrandingImage',
         'windowSplashScreenIconBackgroundColor',
         'postSplashScreenTheme'
     ].forEach(themeKey => {
-        const cdvConfigPrefKey = 'Android' + themeKey.charAt(0).toUpperCase() + themeKey.slice(1);
+        let index = 0;
+        if (themeKey.indexOf(':') !== -1) {
+            index = themeKey.indexOf(':') + 1;
+        }
+        const cdvConfigPrefKey = 'Android' + themeKey.charAt(index).toUpperCase() + themeKey.slice(index + 1);
```

Comment:
Not sure if this is better performance or eaiser to read, but what about this?

```js
const themeKeyDescoped = themeKey.replace('android:', '');
const cdvConfigPrefKey = 'Android' + themeKeyDescoped.charAt(0).toUpperCase() + themeKeyDescoped.slice(1);
```

Reply:
I have removed the if as it's not really necessary, so the code is simpler now, let me know if you still prefer to use the replace.

Reply:
What you have is good. Dont need to use replace.